examples/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Examples of using CouchBackup as a library",
   "dependencies": {
     "aws-sdk": "^2.39.0",
-    "couchbackup": "^2.0.0",
```

Comment:
Without this dependency here does the `require` statement need to be updated to point to the local file or it will it safely pick it up from the local `node_modeules` ahead of any globally installed copy?